jena-arq/src/main/java/org/apache/jena/query/ResultSet.java

```diff
@@ -103,4 +103,8 @@ public default ResultSet materialise() {
     }
 
     public void close();
+
+    default RowSet toRowSet() {
```

Comment:
I prefer `asRowSet` -- `as*` when it is the  same thing, different presentation, and `to*` when it is a created conversion.

c.f. `Dataset::asDatasetGraph`

Reply:
Renamed to `asRowSet`.